streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -406,24 +361,6 @@ private void cleanGlobal(final String intermediateUserTopic) {
         Assert.assertEquals(0, exitCode);
     }
 
-    private void waitUntilUserTopicsAreDeleted() {
```

Comment:
What I was saying is that we can remove the call of `assertInternalTopicsGotDeleted` with `TestUtils.waitForCondition(TopicsDeletedCondition)` where we pass the internalTopics plus the `OUTPUT_TOPIC ..`

Anyways, it is only minor comment so let me know if you prefer this way and I can merge as is.